src/Source/DoctrineODMQuerySourceIterator.php

```diff
@@ -64,7 +70,23 @@ public function __construct(Query $query, array $fields, $dateTimeFormat = 'r')
             }
         }
 
-        $this->dateTimeFormat = $dateTimeFormat;
+        $formatters = array(new ArrayFormatter(), new DateTimeFormatter($dateTimeFormat), new ObjectToStringFormatter());
```

Comment:
Well, why is this deprecated? This particular set of code can stay as is if you ask me. If you don't require per property DateTime formats it should be fine?